core/src/main/java/org/apache/calcite/sql/dialect/SparkSqlDialect.java

```diff
@@ -147,9 +147,34 @@ public SparkSqlDialect(SqlDialect.Context context) {
       case TRIM:
         unparseHiveTrim(writer, call, leftPrec, rightPrec);
         break;
+      case POSITION:
```

Comment:
I noticed `POSITION('a', 'abc')` and `POSITION('a' IN 'abc')` all works in Spark. So we can use 
` case POSITION:
    SqlUtil.unparseFunctionSyntax(SqlStdOperatorTable.POSITION, writer, call, false);`. WDYT?